src/ScottPlot/Settings.cs

```diff
@@ -495,12 +495,22 @@ public void MouseZoomRect(float mouseNowX, float mouseNowY, bool finalize = fals
 
             if (finalize)
             {
-                double x1 = XAxis.Dims.GetUnit(left);
-                double x2 = XAxis.Dims.GetUnit(right);
-                double y1 = YAxis.Dims.GetUnit(bottom);
-                double y2 = YAxis.Dims.GetUnit(top);
-                ZoomRectangle.Clear();
-                AxisSet(x1, x2, y1, y2);
+                foreach (Axis axis in Axes)
+                {
+                    ZoomRectangle.Clear();
```

Comment:
Is there a reason we clear it up here? If possible it would be nice to move it out of the loop.

Reply:
Nope, no reason, I just missed this when moving things around. Solved in 724e892ab66c3111b3dc3cdda5f66a00f7658f9a

Reply:
Looks good, I think only you can resolve this thread.

Reply:
> I think only you can resolve this thread

I can resolve it too 😈

Thanks @BambOoxX! I'll merge this tonight 👍